solidity/contracts/bancorx/BancorX.sol

```diff
@@ -444,7 +450,10 @@ contract BancorX is IBancorX, TokenHolder, ContractRegistryClient {
         // prevReleaseLimit + ((currBlockNumber - prevReleaseBlockNumber) * limitIncPerBlock)
         uint256 currentReleaseLimit =
             prevReleaseLimit.add(((block.number).sub(prevReleaseBlockNumber)).mul(limitIncPerBlock));
-        if (currentReleaseLimit > maxReleaseLimit) return maxReleaseLimit;
+        if (currentReleaseLimit > maxReleaseLimit) {
```

Comment:
Can simplify this to:
```
return Math.min(currentReleaseLimit, maxReleaseLimit);
```

Reply:
This PR is already too big, so let's postpone unrelated old code refactoring (unlike style or formatting)